src/tiledb/cloud/bioimg/helpers.py

```diff
@@ -16,20 +16,22 @@ def get_uris(
     vfs = tiledb.VFS(config=config)
 
     def create_output_path(input_file, output_dir) -> str:
-        return os.path.join(output_dir, os.path.basename(input_file) + f".{output_ext}")
+        filename = os.path.splitext(os.path.basename(input_file))[0]
+        return os.path.join(output_dir, filename + f".{output_ext}")
 
     def iter_paths(sequence) -> Iterator[Tuple]:
         for uri in sequence:
-            yield uri, create_output_path(uri, output_dir)
+            if uri.endswith((".tiff", ".tif", ".tdb")):
```

Comment:
Do we need a constant list? We support more than `tiff` right?

Reply:
We will for now the client calls only the UDF with the OMETIFF converter. We have already implemented an abstraction `from_bioimg` that includes all converters and will auto-recognize based on extension. For now the functionality is being constrained to the above.